purchases/src/test/java/com/revenuecat/purchases/paywalls/PaywallDataTest.kt

```diff
@@ -77,8 +59,6 @@ class PaywallDataTest {
             assertThat(darkColors.accent2?.stringRepresentation).isEqualTo("#FF00FF")
         }
 
-        mockLocale("gl_ES", "glg")
```

Comment:
Oh so we don't need to mock this anymore because we catch in `sharedLanguageCodeWith` and use language to compare right?

Reply:
Well, actually I believe the issues you were running into are mostly fixed by using the `Locale.forLanguageTag` constructor. Before, we were doing `Locale(localeKey)` where `localeKey` was a tag but a language code is expected there. After using that, the tests were fixed without having to mock.

Reply:
You're right! Fixed in https://github.com/RevenueCat/purchases-android/pull/1267/commits/eeef67697e675bce67a0d96dbcea451c86e6175a